symfony/plugins/orangehrmAttendancePlugin/config/routes.yaml

```
@@ -34,6 +34,22 @@ apiv2_employee_latest_attendance_record:
     id: 0
     _api: OrangeHRM\Attendance\Api\EmployeeLatestAttendanceRecordAPI
 
+apiv2_attendance_punch_in_record_overlap:
+  path: /api/v2/attendance/punch-in/records/overlaps
```

```suggestion
  path: /api/v2/attendance/punch-in/overlaps
```
:thinking: what do you think?

---

done